lava/lib/dl/bootstrap/README.md

```diff
@@ -1,6 +1,8 @@
 # Lava-DL Bootstrap
 
-`lava.lib.dl.bootstrap` accelerates rate coded Spiking Neural Network (SNN) training using by dynamically estimating the equivalent ANN transfer function of a spiking layer with a picewise linear model at regular interval and using the ANN equivlent network to train the original SNN. 
+In general ANN-SNN conversion methods for rate based SNN result in high latency of the network. This is because the rate interpretation of a spiking neuron using ReLU acitvation unit breaks down for short inference times. 
```

Comment:
high latency inference

Reply:
Rate based SNNs generated by ANN-SNN conversion methods require long runtimes to achieve high accuracy because the rate-based approximation of the ReLU activation unit breaks down for short runtimes.